FFMpegCore/FFMpeg/FFMpeg.cs

```diff
@@ -92,15 +96,25 @@ await arguments
             return new Bitmap(ms);
         }
 
-        private static (FFMpegArguments, Action<FFMpegArgumentOptions> outputOptions) BuildSnapshotArguments(IMediaAnalysis source, Size? size = null, TimeSpan? captureTime = null)
+        private static (FFMpegArguments, Action<FFMpegArgumentOptions> outputOptions) BuildSnapshotArguments(IMediaAnalysis source, Size? size = null, TimeSpan? captureTime = null, int streamIndex = 0)
         {
             captureTime ??= TimeSpan.FromSeconds(source.Duration.TotalSeconds / 3);
             size = PrepareSnapshotSize(source, size);
 
+            // If user will know about numeration of streams (user passes index of necessary video stream)            
+            int index = source.VideoStreams.Where(videoStream => videoStream.Index == streamIndex).FirstOrDefault().Index;
```

Comment:
Maybe use `.First(videoStream => videoStream.Index == streamIndex)` instead of .Where(..).FirstOrDefault() to provoke a better exception than null reference exception